src/edit_mode/vi/command.rs

```diff
@@ -164,6 +169,7 @@ impl Command {
             Self::PasteAfter => vec![ReedlineOption::Edit(EditCommand::PasteCutBufferAfter)],
             Self::PasteBefore => vec![ReedlineOption::Edit(EditCommand::PasteCutBufferBefore)],
             Self::Undo => vec![ReedlineOption::Edit(EditCommand::Undo)],
+            Self::ChangeTillEnd => vec![ReedlineOption::Edit(EditCommand::CutToLineEnd)],
```

Comment:
This should probably be `ClearToLineEnd`. `CutToLineEnd` will set the clipboard to the deleted content.